include/dump_utils.hpp

```diff
@@ -64,9 +62,17 @@ inline void getValidDumpEntryForAttachment(
                                     "LogServices", "Dump", "Entries", entryID));
             return;
         }
+        dumpType = boost::algorithm::to_lower_copy(entryID.substr(0, pos));
         dumpId = entryID.substr(pos + 1);
     }
 
+    if (dumpType.empty() || entryID.empty())
+    {
+        // Other Types, or BMC or System-variant type dumps
```

Comment:
`// BMC or System-variant type dumps ?`

Reply:
Changed to "Besides of system,resource,sbe,hwdump and host boot dumps"